doc/python/probe_sample.rst

```diff
@@ -5,14 +5,14 @@ Cable cell probing and sampling
 
 .. module:: arbor
 
-Cable cell probe addresses are defined analogously to their counterparts in
+Cable cell probeset addresses are defined analogously to their counterparts in
 the C++ API (see :ref:`cablecell-probes` for details). Sample data recorded
 by the Arbor simulation object is returned in the form of a NumPy array,
 with the first column holding sample times, and subsequent columns holding
 the corresponding scalar- or vector-valued sample.
```

Comment:
So do I understand correctly that in the case of scalar valued probes we get an `(N, 2)` shaped numpy array, and in the case of an `M`-long vector valued probe we'd get an `(N, 1 + M)` shaped array? Might be worth to show 2 examples of that here. Plus, we'd repeat that `K` times for each probe in the probeset?

Reply:
`sim.samples(handle)` will return a list of length `K`. In each element, you'll find a tuple containing `metadata` and `data`, where `metadata` will be a string describing the location, and `data` a `numpy.ndarray`.